src/Components/SigDetailsTable/__tests__/sigDetailsTable.fixtures.js

```diff
@@ -66,3 +71,126 @@ export const mockSigHitsList = {
     ],
   },
 };
+
+export const mockSigDetailsGroupData = {
+  data: {
+    rulesList: [
+      {
+        affectedHostsList: [
+          {
+            groups: [
+              {
+                id: '1234',
+                name: 'workspace1',
+              },
+            ],
+          },
+        ],
+      },
+    ],
+  },
+};
+
+export const sigDetailsTableMockRequestDefault = {
+  query: GET_SIGNATURE_DETAILS_TABLE,
+  variables: {
+    displayName: '',
+    hostGroupFilter: undefined,
+    limit: 10,
+    offset: 0,
+    orderBy: 'LAST_SCAN_DATE_DESC',
+    sigMatchStatusFilter: undefined,
+    ruleName: 'bingo bango bongo',
+  },
+};
+
+export const sigDetailsTableMockRequestStatusNotReviewed = {
+  query: GET_SIGNATURE_DETAILS_TABLE,
+  variables: {
+    displayName: '',
+    hostGroupFilter: undefined,
+    limit: 10,
+    offset: 0,
+    orderBy: 'LAST_SCAN_DATE_DESC',
+    sigMatchStatusFilter: ['NOT_REVIEWED'],
+    ruleName: 'bingo bango bongo',
```

Comment:
It might be good to have a const with all common variable, something like `defaultVariables` to spread in here and only set the variable that is different here. Like:

```js
 variables: {
     ...defaultVariables,
     sigMatchStatusFilter: ['NOT_REVIEWED'],
 }
````

It's less code and easier to identify how all these consts differ from each other.